client/pages/addcard.jsx

```diff
@@ -33,26 +33,27 @@ export default class AddCard extends React.Component {
         this.setState({ question: '', answer: '' });
       })
       .catch(err => console.error(err));
+    location.href = `/#view-cards?deckName=${this.props.deckName}?deckId=${deckId}`;
```

Comment:
It's best to wait until the fetch actually completes, so you'll want to move this hash update to inside your `.then()` callback.

Reply:
Makes sense. I moved it here. The other thing I was wondering is later if I add in a loading modal that I switch over right away but display a loading screen until the fetch completes? That way the user moves right away instead of sitting on the add screen until the fetch is completed (this is more of a later problem)

![image](https://user-images.githubusercontent.com/109567499/200909540-7016dc80-e7bc-46bc-8c13-f740be05634e.png)